content/docs/ops/security/keys-and-certs/index.md

```diff
@@ -179,3 +181,142 @@ Certificate:
          0c:33:42:f0:b6:f5:1c:0d:fb:70:65:aa:93:3e:ca:0e:58:ec:
          8e:d5:d0:1e
 {{< /text >}}
+
+## How Citadel decides whether to create Service Account secrets
+
+### Overview
+
+When a Citadel instance notices that a `ServiceAccount` was created in a namespace, it needs to decide whether it should generate an `istio.io/key-and-cert` secret for that `ServiceAccount`. In order to make that decision, there are three relevant values (note: there can be multiple Citadel instances deployed in a single cluster, and the following targeting rules are applied to each instance):
+
+1. **`ca.istio.io/env` namespace label**: *string valued* label which should be pointed at the namespace of the desired Citadel instance
+
+1. **`ca.istio.io/override` namespace label**: *boolean valued* label which overrides all other configurations and forces all Citadel instances either to target or ignore a namespace
+
+1. [**`enableNamespacesByDefault` security configuration**](/docs/reference/config/installation-options/#security-options): default behavior if no labels are found on the `ServiceAccount's` namespace
+
+From these three values, the decision process mirrors that of the [`Sidecar Injection Webhook`](/docs/ops/setup/injection/). The detailed behavior is that:
+
+* If `ca.istio.io/override` exists and is valid, then follow what it tells us (generate key/cert secrets for workloads if `true`, don't if `false`)
+* If not, then check `ca.istio.io/env: "ns-foo"`. The Citadel instance in namespace `ns-foo` will be responsible for generating key/cert secrets for workloads
+* If `ca.istio.io/env` is nonexistent or invalid, follow the `enableNamespacesByDefault` configuration value
```

Comment:
I would prefer to call out this as a helm flag. Or refer to other docs, see what terminology we use when referring helm flag?